### PR TITLE
Attempt to code for issue #61

### PR DESCRIPTION
**Pull request by picapi**
https://github.com/SteamDatabase/steamSummerMinigame/issues/61

Attempted to code this, it's my first attempt at this kind of javascript but it seems to work, if anyone wants to test it that would be great!

### CODE REVIEW

**Comment by LeoVerto** [comment]
Please do not increment the version in your PR.

**Comment by picapi** [comment]
Literally noticed that as you said that :/ fixed it now

File: autoPlay.user.js [hunk 1]
```diff
@@ -203,7 +204,7 @@ function MainLoop() {
 		useGoldRainIfRelevant();
 		useMetalDetectorIfRelevant();
 		useCrippleMonsterIfRelevant();
-
+		AutoUseMedicIfRelevant()
```
Comment (LeoVerto) [inline autoPlay.user.js:207]: The function name should be changed to "useMedicIfRelevant" to be consistent with the other functions.
You also forgot a semicolon here (welcome to JS :P).

**Comment by iakat** [comment]
Please squash the commits once you're done.

**Comment by ozerugae** [comment]
@picapi does it take into account whether a cooldown is present in the lane?

File: autoPlay.user.js [hunk 2]
```diff
@@ -192,7 +193,7 @@ function MainLoop() {
 		isAlreadyRunning = true;
 
 		goToLaneWithBestTarget();
-
+		
 		useGoodLuckCharmIfRelevant();
 		useMedicsIfRelevant();
```
Comment (iakat) [inline autoPlay.user.js:198]: This is actually already implemented. #61

**Pull request closed without merging**